### PR TITLE
Improve description of database and add - import part

### PR DESCRIPTION
**Pull request by gpetit**
Improve description of database : add a schema

Improve description for "add - import" part (thx @nicolas-f ) as asked in orbisgis/orbisgis#1193

### CODE REVIEW

File: docs/users/quickstart.rst [hunk 1]
```diff
@@ -71,7 +71,11 @@ Database type:
 * H2GIS_SERVER: remote H2GIS db,
 * POSTGIS: remote PostGIS db.
 
+The figure below summarizes this philosophy:
```
Comment (ebocher) [inline docs/users/quickstart.rst]: architecture ? or usage ?

File: docs/users/quickstart.rst [hunk 2]
```diff
@@ -87,8 +91,24 @@ Load data
 
 To load data, just make a right-click in the :ref:`geocatalog` and choose "Add" or "Import".
 
-* Add: link to a file (vector, alphanumeric, ...),
-* Import: import a file and convert it into a table.
+* Add: Link a new database table to a file (vector, alphanumeric, ...),
+	* Pro:
+		* Immediate table creation,
```
Comment (ebocher) [inline docs/users/quickstart.rst]: Direct link to the data

File: docs/users/quickstart.rst [hunk 3]
```diff
@@ -87,8 +91,24 @@ Load data
...
+		* Does not take more disk space,
```
Comment (ebocher) [inline docs/users/quickstart.rst]: remove more

File: docs/users/quickstart.rst [hunk 4]
```diff
@@ -87,8 +91,24 @@ Load data
...
+		* Can create spatial index.
+	* Cons:
+		* Read-only table,
+		* Cannot create alpha-numeric index,
+		* Only compatible with h2 database.
```
Comment (ebocher) [inline docs/users/quickstart.rst]: Nope we can connect two remote Postgis tables with dblink https://www.postgresql.org/docs/9.2/static/dblink.html

File: docs/users/quickstart.rst (hunk 1 above)
Comment (gpetit) [reply docs/users/quickstart.rst]: Ok for usage

File: docs/users/quickstart.rst (hunk 4 above)
Comment (gpetit) [reply docs/users/quickstart.rst]: Ok so I remove the last line

**Comment by nicolas-f** [comment]
The add function does not support dblink

**Comment by nicolas-f** [comment]
And dblink does not connect to a file but to another database

**Comment by nicolas-f** [comment]
So the Add function is only compatible with local h2 database

**Comment by ebocher** [comment]
@nicolas-f 
You are right we will support dblink and datalink in the future from a WPS script

**Comment by ebocher** [comment]
Nice

**Pull request merged**